Animation/animation2/style2.css

```
@@ -0,0 +1,60 @@
+.container{
+    width: 400px;
+    height: 400px;
+    background-color: rgb(43, 88, 113);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+h1{
+    color: rgb(251, 251, 251);
+    text-align: center;
+    margin: 16px;
+    animation: heading 2s ease-in-out 0.8s 1;
+}
```

As recomanda a textul deja sa inceapa din afara ca apoi sa vina incet spre mijloc:

```suggestion
    animation: heading 2s ease-in-out 0.8s forwards;
    transform: translateY(-400px);
```